docker/intel-dgpu-driver.Dockerfile

```diff
@@ -33,4 +33,4 @@ COPY --from=builder /etc/driver-toolkit-release.json /etc/
 COPY --from=builder /lib/modules/$KERNEL_FULL_VERSION/ /opt/lib/modules/$KERNEL_FULL_VERSION/
 COPY --from=builder /build/intel-gpu-firmware/firmware/ /firmware/i915/
 
-RUN depmod -b /opt
```

Comment:
we should keep this line. 
RUN depmod -b /opt
is useful actually. :-)

Reply:
Line 36 which contains the depmod is not actually removed. It looks like it is removed but what actually happened is a newline was not present before on line 36. With the PR, a newline is being added to line 36. Take a look at the git diff again. :)